raphtory/src/python/graph/graph.rs

```diff
@@ -566,6 +724,33 @@ impl PyGraph {
         )
     }
 
+    /// Load node properties from a parquet file.
+    ///
+    /// Arguments:
+    ///     parquet_path (str): Parquet file or directory of Parquet files path containing node information.
```

Comment:
Not for now - but I think this function should have the ability to set node_type - will turn this into a ticket